src/theme/index.hbs

```diff
@@ -6,6 +6,7 @@
         <meta content="text/html; charset=utf-8" http-equiv="Content-Type">
         <meta name="description" content="{{ description }}">
         <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="theme-color" content="#ffffff" />
```

Comment:
`#ffffff` is the default value, but putting it in the markup avoids having to manage two code paths (adding the tag and updating it).